src/code/code_800EC960.c

```diff
@@ -2855,21 +2870,21 @@ void func_800F3ED4(void) {
     for (i = 0; i < 16; i++) {
```

Comment:
```suggestion
    for (i = 0; i < ARRAY_COUNT(D_8016B8B8); i++) {
```

Reply:
Not sure about this actually: what we really should have is a define for the number of channels in a sequence player. Using ARRAY_COUNT makes it look like this code works regardless of the array size, but then the `D_8016B8B8[0xD].unk_0C = 0;` looks out of place. (I have no idea what that line is doing there fwiw...)

Reply:
alright a define sounds better than the array_count macro so scratch that

for `D_8016B8B8[13]` I guess it would be a job for a future `assert(13 < NUM_CHANNELS)`, like many similar things everywhere in the code